### PR TITLE
Add two unsupported test cases

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: py2many/inference.py [hunk 1]
```diff
@@ -404,6 +404,16 @@ def visit_BinOp(self, node):
                 node.annotation = ast.Name(id="float")
             return node
 
+        # Container multiplication
```
Comment (adsharma) [inline py2many/inference.py:407]: some of this is covered by `LEGAL_COMBINATIONS` further below. Perhaps moving the two pieces of code closer together could help the flow.

Also, `1 + 2j` errors out here. Perhaps make it legal while you're here?

Comment (jayvdb) [reply py2many/inference.py:415]: Moved as requested.  And created https://github.com/adsharma/py2many/issues/379 for the unrelated `complex` set of binops, as tests also need creating for them.

Comment (adsharma) [reply py2many/inference.py:417]: I see two instances of this code in the diff. Was it intentional?

File: py2many/inference.py [hunk 2]
```diff
@@ -435,26 +445,32 @@ def visit_BinOp(self, node):
                     return node
             node.annotation = left
             return node
-        else:
```
Comment (adsharma) [inline py2many/inference.py:438]: This code was covering `i8-i64 binop float` case. Perhaps didn't have test coverage.

File: py2many/inference.py (hunk 1 above)
Comment (jayvdb) [reply py2many/inference.py]: No. Resulted from multiple rebases.

File: py2many/inference.py (hunk 2 above)
Comment (jayvdb) [reply py2many/inference.py:438]: I believe no code path was changed here.  I only removed the unnecessary dangling else.

**Pull request merged**